bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
Argh, I was looking at the wrong page (developer/index). Then yes, we need a tag, and to move these strings in a shared file if we start using these strings in other pages.